subworkflows/local/align.nf

```diff
@@ -31,9 +32,17 @@ workflow ALIGN {
         ch_bwamem2_bai    = Channel.empty()
         ch_sentieon_bai   = Channel.empty()
 
+        if (!params.skip_germlinecnvcaller) {
+            FASTP(ch_reads, [], false, false)
+            ch_input_reads = FASTP.out.reads
```

Comment:
```suggestion
            ch_reads = FASTP.out.reads
```

Reply:
Is it possible to just overwrite the original channel with the output from the fastp?

Reply:
I am unsure, that's why I created a new one. I can try it out

Reply:
I tried and it worked :)